### ISSUE TITLE
hosc-0.18 fails to install on older distros

### ISSUE THREAD

**Issue by cleary** [cleary/ansible-tidalcycles#1]
Ref: https://github.com/rd--/hosc/pull/2

```
 Building hosc-0.18...
Preprocessing library hosc-0.18...
[ 1 of 18] Compiling Sound.OSC.Wait   ( Sound/OSC/Wait.hs, dist/build/Sound/OSC/Wait.o )
[ 2 of 18] Compiling Sound.OSC.Coding.Convert ( Sound/OSC/Coding/Convert.hs, dist/build/Sound/OSC/Coding/Convert.o )
[ 3 of 18] Compiling Sound.OSC.Time   ( Sound/OSC/Time.hs, dist/build/Sound/OSC/Time.o )
[ 4 of 18] Compiling Sound.OSC.Datum  ( Sound/OSC/Datum.hs, dist/build/Sound/OSC/Datum.o )
[ 5 of 18] Compiling Sound.OSC.Packet ( Sound/OSC/Packet.hs, dist/build/Sound/OSC/Packet.o )
[ 6 of 18] Compiling Sound.OSC.Transport.FD ( Sound/OSC/Transport/FD.hs, dist/build/Sound/OSC/Transport/FD.o )
[ 7 of 18] Compiling Sound.OSC.Transport.Monad ( Sound/OSC/Transport/Monad.hs, dist/build/Sound/OSC/Transport/Monad.o )
[ 8 of 18] Compiling Sound.OSC.Coding.Cast ( Sound/OSC/Coding/Cast.hs, dist/build/Sound/OSC/Coding/Cast.o )
[ 9 of 18] Compiling Sound.OSC.Coding.Byte ( Sound/OSC/Coding/Byte.hs, dist/build/Sound/OSC/Coding/Byte.o )
[10 of 18] Compiling Sound.OSC.Coding.Decode.Base ( Sound/OSC/Coding/Decode/Base.hs, dist/build/Sound/OSC/Coding/Decode/Base.o )
[11 of 18] Compiling Sound.OSC.Coding.Decode.Binary ( Sound/OSC/Coding/Decode/Binary.hs, dist/build/Sound/OSC/Coding/Decode/Binary.o )
[12 of 18] Compiling Sound.OSC.Coding.Encode.Base ( Sound/OSC/Coding/Encode/Base.hs, dist/build/Sound/OSC/Coding/Encode/Base.o )
[13 of 18] Compiling Sound.OSC.Coding.Encode.Builder ( Sound/OSC/Coding/Encode/Builder.hs, dist/build/Sound/OSC/Coding/Encode/Builder.o )

Sound/OSC/Coding/Encode/Builder.hs:33:36: error:
    • Variable not in scope:
        (<>) :: B.Builder -> B.Builder -> B.Builder
    • Perhaps you meant one of these:
        ‘<$>’ (imported from Prelude), ‘*>’ (imported from Prelude),
        ‘<$’ (imported from Prelude)

Sound/OSC/Coding/Encode/Builder.hs:37:33: error:
    • Variable not in scope:
        (<>) :: B.Builder -> B.Builder -> B.Builder
    • Perhaps you meant one of these:
        ‘<$>’ (imported from Prelude), ‘*>’ (imported from Prelude),
        ‘<$’ (imported from Prelude)

Sound/OSC/Coding/Encode/Builder.hs:42:55: error:
    • Variable not in scope: (<>) :: B.Builder -> B.Builder -> t0
    • Perhaps you meant one of these:
        ‘<$>’ (imported from Prelude), ‘*>’ (imported from Prelude),
        ‘<$’ (imported from Prelude)

Sound/OSC/Coding/Encode/Builder.hs:43:26: error:
    • Variable not in scope: (<>) :: t0 -> B.Builder -> B.Builder
    • Perhaps you meant one of these:
        ‘<$>’ (imported from Prelude), ‘*>’ (imported from Prelude),
        ‘<$’ (imported from Prelude)
cabal: Leaving directory '/tmp/cabal-tmp-14650/hosc-0.18'
Building vector-0.12.1.2...
Installed distributive-0.6.2
Downloading comonad-5.0.6...
Configuring comonad-5.0.6...
Building comonad-5.0.6...
Installed comonad-5.0.6
Downloading bifunctors-5.5.7...
Configuring bifunctors-5.5.7...
Building bifunctors-5.5.7...
Installed bifunctors-5.5.7
Installed vector-0.12.1.2
cabal: Error: some packages failed to install:
hosc-0.18 failed during the building phase. The exception was:
ExitFailure 1
tidal-1.6.0 depends on hosc-0.18 which failed to install.
```

**Comment by cleary** [comment]
Issue was in the wrong repo, see:

https://github.com/cleary/ansible-tidalcycles-base/commit/7003332b07cbc4024a74e13eec838870484eeeba

https://github.com/cleary/ansible-tidalcycles-base/commit/c83a5a431a5ac8cbfebe8c463360c85ee4885bd0
